packages/phone-number-privacy/signer/src/database/models/domain.ts

```diff
@@ -0,0 +1,20 @@
+export const DOMAINS_TABLE = 'domains'
+export enum DOMAINS_COLUMNS {
+  domain = 'domain',
+  counter = 'counter',
+  timer = 'timer',
+  disabled = 'disabled',
```

Comment:
I'm not sure that it does. 
- all domains will require the 'disabled' column
- Many domains will have some timestamp that they store. 
- Many domains will have a counter to track how many requests have been made
- New columns can be added as needed

Reply:
I will note that disabling a domain is not a required feature of all domains, and unauthenticated domains will most likely not implement it as there would be no way to prevent DoS attacks.

I don't share the intuition that these fields will be shared among many domains, but also don't see this as a major issue. It's true that we can implement new domains in a shared table where each uses a subset of the columns, and there is no need to change this approach unless things get out of hand. If that happens, we would need a database migration, but that doesn't seem like a major issue.

TL;DR happy to defer to y'all's opinion on this.

Reply:
Ahh I misunderstood the CIP wrt disabling domains. I agree we can keep this as is for simplicity now and do a db migration when / if that comes up. Good thing to have on our radar for sure though